### PR TITLE
Update the Get started doc

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

This PR is related with https://github.com/netdata/documentation/issues/22, it includes:

- [X] Added a section called "Install Netdata", providing a very quick route to installing and claiming from the Cloud
- [X] The `wget` and `curl` options were put into tabs that Docusaurus provides
- [X] Fixed broken links with the proper (from now on) absolute github links
- [X] A note has been added to instruct the user to change the placeholders if he intends to claim to the Cloud
- [X] Find a way to not have absolute raw links inside the install option boxes 👉 https://github.com/netdata/learn/pull/1292

What else could be changed:

- Do we need the top section "Get started with Netdata"? (Same information should be living at the introduction document)
- ~~Should we rename the file's title to "Install Netdata"?~~ Already done due to naming conflict with the category this file lives at Learn
- Should we add another box in the "other installation options" for the "Install Netdata with kickstart.sh" page? (we have a link to it, but as a reference to the kickstart script)

### CODE REVIEW

File: docs/get-started.mdx [hunk 1]
```diff
@@ -23,47 +23,72 @@ Netdata runs permanently on all your physical/virtual servers, containers, cloud
 runs on Linux distributions (Ubuntu, Debian, CentOS, and more), container/microservice platforms (Kubernetes clusters,
 Docker), and many other operating systems (FreeBSD, macOS), with no `sudo` required.
 
+To install Netdata in minutes on your platform:
+
+1. Visit https://app.netdata.cloud/
```
Comment (cakrit) [inline docs/get-started.mdx:28]:
```suggestion
1. Sign up to https://app.netdata.cloud/
```

File: docs/get-started.mdx [hunk 2]
```diff
@@ -23,47 +23,72 @@ Netdata runs permanently on all your physical/virtual servers, containers, cloud
...
+2. You will be presented with an empty space, and a prompt with the install command for each platform
+3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it
+
+Upon installation completing successfully, you should be able to see the node live in your Netdata Space!
```
Comment (cakrit) [inline docs/get-started.mdx:32]:
```suggestion
Upon installation completing successfully, you should be able to see the node live in your Netdata Space!

Continue reading for more advanced instructions and installation options.
```

File: docs/get-started.mdx [hunk 3]
```diff
@@ -75,35 +100,33 @@ Where you go from here is based on your use case, immediate needs, and experienc
...
-community, but you may want to [edit alarms](/docs/monitor/configure-alarms.md) or [enable
-notifications](/docs/monitor/enable-notifications.md) to customize your Netdata experience.
+community, but you may want to [edit alarms](https://github.com/netdata/netdata/blob/master/docs/monitor/configure-alarms.md) or 
+[enable notifications](https://github.com/netdata/netdata/blob/master/docs/monitor/enable-notifications.md) to customize your Netdata experience.
 
 ### Need to monitor multiple nodes in one place?
 
-For robust multi-node monitoring from a single interface, consider [Netdata
-Cloud](https://learn.netdata.cloud/docs/cloud), which streams, aggregates, and visualizes metrics from any number of
+For robust multi-node monitoring from a single interface, consider [Netdata Cloud](https://app.netdata.cloud/), which streams, aggregates, and visualizes metrics from any number of
```
Comment (cakrit) [inline docs/get-started.mdx:126]: Delete this. Our main instruction is WITH the cloud.

File: docs/get-started.mdx [hunk 4]
```diff
@@ -75,35 +100,33 @@ Where you go from here is based on your use case, immediate needs, and experienc
...
 nodes. It's all the same out-of-the-box, zero-configuration functionality of the open-source monitoring agent, but for
 any number of distributed nodes, _entirely for free_.
 
 There is an alternative for those who aren't interested in using Netdata Cloud, albeit with some required configuration.
-Each node can [stream](/streaming/README.md) its metrics to any other node, and the default
-[registry](/registry/README.md) is configurable to create a private "network" of Netdata dashboards.
+Each node can [stream](https://github.com/netdata/netdata/blob/master/streaming/README.md) its metrics to any other node, and the default
+[registry](https://github.com/netdata/netdata/blob/master/registry/README.md) is configurable to create a private "network" of Netdata dashboards.
```
Comment (cakrit) [inline docs/get-started.mdx:132]: The default registry isn't really worth mentioning here, very edge case.

File: docs/get-started.mdx [hunk 5]
```diff
@@ -75,35 +100,33 @@ Where you go from here is based on your use case, immediate needs, and experienc
...
 
 ### Alarms & notifications
 
 Netdata comes with hundreds of preconfigured alarms, designed by our monitoring gurus in parallel with our open-source
-community, but you may want to [edit alarms](/docs/monitor/configure-alarms.md) or [enable
-notifications](/docs/monitor/enable-notifications.md) to customize your Netdata experience.
+community, but you may want to [edit alarms](https://github.com/netdata/netdata/blob/master/docs/monitor/configure-alarms.md) or 
+[enable notifications](https://github.com/netdata/netdata/blob/master/docs/monitor/enable-notifications.md) to customize your Netdata experience.
 
 ### Need to monitor multiple nodes in one place?
```
[the remark this answers is not in the document]
Comment (cakrit) [reply docs/get-started.mdx:124]: Both securing and replication are strongly recommended. We'll probably create a dedicated page for production readiness, but not now.

**Review by cakrit — CHANGES_REQUESTED** [review]
See individual comments and handle them. Haven't looked at the widget yet.

File: docs/get-started.mdx [hunk 6]
```diff
@@ -75,35 +102,27 @@ Where you go from here is based on your use case, immediate needs, and experienc
...
-For robust multi-node monitoring from a single interface, consider [Netdata
-Cloud](https://learn.netdata.cloud/docs/cloud), which streams, aggregates, and visualizes metrics from any number of
-nodes. It's all the same out-of-the-box, zero-configuration functionality of the open-source monitoring agent, but for
-any number of distributed nodes, _entirely for free_.
+### Make your deployment production ready
 
-There is an alternative for those who aren't interested in using Netdata Cloud, albeit with some required configuration.
-Each node can [stream](/streaming/README.md) its metrics to any other node, and the default
-[registry](/registry/README.md) is configurable to create a private "network" of Netdata dashboards.
+Both [securing](https://github.com/netdata/netdata/blob/master/docs/configure/secure-nodes.md) and [replication](https://github.com/netdata/netdata/blob/master/streaming/README.md) are strongly recommended.
```
Comment (cakrit) [inline docs/get-started.mdx:128]:
```suggestion
Both [securing Netdata](https://github.com/netdata/netdata/blob/master/docs/configure/secure-nodes.md) and [setting up replication](https://github.com/netdata/netdata/blob/master/streaming/README.md) are strongly recommended.
```

**Review by cakrit — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**